spec/OpenLineage.yml

```diff
@@ -0,0 +1,250 @@
+openapi: 3.0.2
+info:
+  title: OpenLineage
+  version: 0.0.1
+  description: OpenLineage is an open source **lineage and metadata collection API** for the data ecosystem.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+paths:
+  /events:
+    post:
+      summary: Updates the state of a run
+      description: Updates a run state for a job.
+      operationId: postRunStateUpdate
+      tags:
+        - OpenLineage
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RunStateUpdates'
+      responses:
+        '200':
+          description: OK
+
+components:
+  schemas:
+    RunStateUpdates:
+      type: array
+      items:
+        $ref: '#/components/schemas/RunStateUpdate'
+
+    RunStateUpdate:
+      type: object
+      properties:
+        runId:
+          description: the unique identifier for that run
+          type: string
+          format: uuid
+        jobId:
+          $ref: '#/components/schemas/JobId'
+        transition:
+          description: transitions the run state
+          type: string #TODO: add enum
```

Comment:
Defining this an an enum may be limiting since systems have a wide array of transition states

Reply:
This is a great point. We do need standard transitions and a mechanism to extend them with custom ones.
Possibly this could be achieved with prefixes (similar to adding facets)